### PR TITLE
Remove support of releases prior to Jewel.

### PR DESCRIPTION
**Pull request by guits**
According to #1216, we need to simply the code by removing the
support of anything before Jewel.

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

**Comment by leseb** [comment]
Thanks @guits, can you add a task to check early if systemd is not present we fail? Something in https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-common/tasks/checks/check_system.yml would be nice.

**Comment by guits** [comment]
sure, i will

File: roles/ceph-common/tasks/facts.yml [hunk 1]
```diff
@@ -70,81 +57,36 @@
...
-    activate_file_mode: "0644"
-  when: ceph_release_num.{{ ceph_release }} < ceph_release_num.infernalis
-
-- set_fact:
-    rbd_client_directory_owner: root
-  when:
-    - ceph_release_num.{{ ceph_release }} < ceph_release_num.infernalis
-    - rbd_client_directory_owner is not defined
-      or not rbd_client_directory_owner
+#- set_fact:
```
Comment (andrewschoen) [inline roles/ceph-common/tasks/facts.yml]: Did you mean to remove these?

Comment (guits) [inline roles/ceph-common/tasks/facts.yml]: yes, i just commented these for some tests

**Comment by leseb** [comment]
@guits are you done with your changes? Are you still working on this? Should we do a final review?
Thanks!

**Comment by guits** [comment]
I'm almost done with the changes, still working on this, actually, i'm just
having a look at the infrastructure-playbooks because there are some
changes to bring in here too.
I think you can start to review the main part of the playbook, I'll let you
know when i'll be ok with the infrastructure-playbooks if you want?


*Guillaume Abrioux*Technical Cloud Consultant


*PHONE : *+33 (0)1 49 70 96 2 <callto:+33%20%280%291%2049%2070%2086%2026>1
*ADDRESS :* 11 bis rue roquépine, Paris 75008

2017-01-25 13:27 GMT+01:00 Sébastien Han <notifications@github.com>:

**Comment by leseb** [comment]
Yes please, let us now when you're done so we can do a final review :). Merci !

**Comment by guits** [comment]
@leseb I just pushed some modifications, I think you can start to do a review

**Comment by guits** [comment]
@leseb: any thoughts?

**Comment by leseb** [comment]
@guits let's keep `rolling_update.yml `outside of this patch please :)

**Comment by guits** [comment]
@leseb: done

**Comment by leseb** [comment]
Please rebase :)

File: roles/ceph-common/handlers/main.yml [hunk 2]
```diff
@@ -46,81 +20,23 @@
...
-
-- name: restart ceph rgws on red hat
-  command: /etc/init.d/ceph-radosgw restart
-  when:
-    - socketrgw.rc == 0
-    - ansible_os_family == 'RedHat'
-    - rgw_group_name in group_names
-    - ceph_release_num.{{ ceph_release }} < ceph_release_num.infernalis
 
 - name: restart ceph rgws with systemd
```
Comment (leseb) [inline roles/ceph-common/handlers/main.yml]: I think we can remove the "with systemd". Don't forget to change `generate_ceph_conf.yml` with the new name as well.

File: roles/ceph-mds/tasks/pre_requisite.yml [hunk 3]
```diff
@@ -39,68 +39,14 @@
 - name: set mds key permissions
   file:
     path: /var/lib/ceph/mds/{{ cluster }}-{{ mds_name }}/keyring
-    mode: "{{ key_mode }}"
-    owner: "{{ key_owner }}"
-    group: "{{ key_group }}"
+    mode: "0600"
```
Comment (leseb) [inline roles/ceph-mds/tasks/pre_requisite.yml]: for consistency, can you move the `mode` after the `group`?

File: roles/ceph-mon/tasks/ceph_keys.yml [hunk 4]
```diff
@@ -112,9 +111,9 @@
 - name: set keys permissions
   file:
     path: "{{ item }}"
-    mode: "{{ key_mode }}"
-    owner: "{{ key_owner }}"
-    group: "{{ key_group }}"
+    mode: "0600"
```
Comment (leseb) [inline roles/ceph-mon/tasks/ceph_keys.yml]: for consistency, can you move the `mode` after the `group`?

File: roles/ceph-mon/tasks/deploy_monitors.yml [hunk 5]
```diff
@@ -24,47 +24,29 @@
 - name: set initial monitor key permissions
   file:
     path: /var/lib/ceph/tmp/keyring.mon.{{ monitor_name }}
-    mode: "{{ key_mode }}"
-    owner: "{{ key_owner }}"
-    group: "{{ key_group }}"
+    mode: "0600"
```
Comment (leseb) [inline roles/ceph-mon/tasks/deploy_monitors.yml]: for consistency, can you move the `mode` after the `group`?

File: roles/ceph-rbd-mirror/tasks/docker/dirs_permissions.yml [hunk 6]
```diff
@@ -33,4 +8,3 @@
     mode: "0755"
```
Comment (leseb) [inline roles/ceph-rbd-mirror/tasks/docker/dirs_permissions.yml]: I think we need `owner` and `group` also

File: roles/ceph-restapi/tasks/docker/dirs_permissions.yml [hunk 7]
```diff
@@ -1,31 +1,5 @@
...
-    - /etc/ceph/
-    - /var/lib/ceph/bootstrap-osd
-    - /var/lib/ceph/bootstrap-mds
-    - /var/lib/ceph/bootstrap-rgw
-  when: not after_hammer
-
-- name: create bootstrap directories (after hammer)
+- name: create bootstrap directories
   file:
     path: "{{ item }}"
```
Comment (leseb) [inline roles/ceph-restapi/tasks/docker/dirs_permissions.yml]: I think we need `owner` and `group` also

File: roles/ceph-rgw/tasks/docker/dirs_permissions.yml [hunk 8]
```diff
@@ -1,31 +1,5 @@
...
-    mode: "0755"
-  with_items:
-    - /etc/ceph/
-    - /var/lib/ceph/bootstrap-rgw
-  when: not after_hammer
-
-- name: create bootstrap directories (after hammer)
+- name: create bootstrap directories
   file:
     path: "{{ item }}"
```
Comment (leseb) [inline roles/ceph-rgw/tasks/docker/dirs_permissions.yml]: I think we need `owner` and `group` also

File: roles/ceph-rgw/tasks/pre_requisite.yml [hunk 9]
```diff
@@ -33,52 +33,14 @@
...
   when: cephx
 
-- name: set rados gateway key permissions (for or after the infernalis release)
+- name: set rados gateway key permissions
   file:
     path: /var/lib/ceph/radosgw/{{ cluster }}-rgw.{{ ansible_hostname }}/keyring
-    mode: "{{ key_mode }}"
-    owner: "{{ key_owner }}"
-    group: "{{ key_group }}"
+    mode: "0600"
```
Comment (leseb) [inline roles/ceph-rgw/tasks/pre_requisite.yml]: for consistency, can you move the `mode` after the `group`?

**Comment by guits** [comment]
jenkins test ansible2.2-purge_cluster

**Pull request merged**